account-keys/slip10/src/lib.rs

```diff
@@ -73,22 +69,32 @@ impl From<[u8; 32]> for Slip10Key {
     }
 }
 
-/// A common interface for constructing a Slip10Key at a particular path from
-/// existing entropy
-pub trait Slip10KeyGenerator {
-    /// The type of error, if any, to be returned if it occurs
-    type Error: Display;
+/// A default derivation of the Slip10Key from a Mnemonic.
+///
+/// This is equivalent to calling `mnemonic.derive_slip10_key(0)`.
+impl From<Mnemonic> for Slip10Key {
+    fn from(src: Mnemonic) -> Slip10Key {
+        src.derive_slip10_key(0)
+    }
+}
 
-    /// Derive a slip10 key for the given path from the current object
-    fn derive_slip10_key(self, path: &[u32]) -> CoreResult<Slip10Key, Self::Error>;
+/// A common interface for constructing a Slip10Key for MobileCoin given an
+/// account index.
+pub trait Slip10KeyGenerator {
+    /// Derive a MobileCoin SLIP10 key for the given account from the current
+    /// object
+    fn derive_slip10_key(self, account_index: u32) -> Slip10Key;
 }
 
+const USAGE_BIP44: u32 = 44;
```

Comment:
Let's add a comment somewhere linking to https://github.com/bitcoin/bips/blob/master/bip-0044.mediawiki ?